packages/components/src/theme/form.scss

```diff
@@ -47,6 +47,10 @@
     text-overflow: ellipsis;
 }
 
+.text__wrap {
+    word-wrap: anywhere;
```

Comment:
I decided to give this a try in a couple different browsers...and of course one was different. In Safari, this CSS rule doesn't work. It looks like "anywhere" isn't fully supported as an option for word-wrap. I think what we'll want is `break-word` (see https://www.w3schools.com/cssref/css3_pr_word-wrap.asp)